src/libraries/System.DirectoryServices.AccountManagement/src/System/DirectoryServices/AccountManagement/Utils.cs

```diff
@@ -811,5 +817,58 @@ internal static bool IsMachineDC(string computerName)
                     Interop.Dsrole.DsRoleFreeMemory(dsRoleInfoPtr);
             }
         }
+
+        //
+        // The sentinel SID were placed in the domain SID range S-1-5-21-X-Y-Z-R with R < 512 because the existing domain controllers would always filter those SIDs out at boundaries.
+        // That way, the sentinel SID which says the claims or compound data is safe to consume would be removed should the claims or compound PAC ever pass through a domain controller
+        // that did not know how to apply security checks. S-1-5-21-X-Y-Z-R means that the SID belongs to a domain(including the local account domain) unless X=Y=Z=0 in which
+        // case it�s a sentinel SID, a special type of pseudo-object that can�t be interpreted in isolation.
```

Comment:
NIT: Non ascii quotes
```suggestion
        // case it's a sentinel SID, a special type of pseudo-object that can't be interpreted in isolation.
```